src/scss/_palette.scss

```diff
@@ -19,68 +19,78 @@
 ///
 /// You can have as many spaces between the colour name and the colour code
 /// as you like (so you can line them up neatly), and you *MUST* finish the
-/// line with a comma (except the final entry, which should not have a comma)
+/// line with a comma.
 ///
 /// @type map
 $o-colors-palette: (
 	// Non-colour CSS values
-	transparent:           transparent,
-	inherit:               inherit,
+	transparent:             transparent,
+	'transparent':           transparent,
+	'inherit':               inherit,
 
 	// Primary Palette
-	pink:                  #fff1e0,
-	black:                 #000000,
-	white:                 #ffffff,
-	blue:                  #2e6e9e,
-	dark-blue:             #275e86,
-	claret:                #9e2f50,
-	orange:                #d66d06,
+	pink:                    #fff1e0, // v2
+	'pink':                  #fff1e0, // v3
+	black:                   #000000, // v2
+	'black':                 #000000, // v3
+	white:                   #ffffff, // v2
+	'white':                 #ffffff, // v3
+	blue:                    #2e6e9e, // v2
+	'blue':                  #2e6e9e, // v3
+	'dark-blue':             #275e86,
```

Comment:
Because in CSS, `dark-blue` is not a color (`DarkBlue` is, though) so Sass will interpret it as a string regardless.